tool/viewmodel/state.py

```diff
@@ -185,6 +185,22 @@ def currentOutcome(self) -> Union[None, ComparisonOutcome]:
         except IndexError:
             return None
 
+    def finish(self):
+        if not self.all_compared:
+            raise ToolInvalidState(
+                "Tried to finish Nodes comparison without comparing all nodes"
+            )
```

Comment:
If it's not too complicated, rather than failing we could give a warning that it's incomplete, and keep all un-reviewed nodes in the network as if they had been skipped or marked 'not same'? That way the user could save the partially finished layer and come back to it later (I think it would work like that?)

Reply:
This is now implemented